samples: sensor: thermometer: extend sample to support frdm_rw612

dipakgmx opened the pull request:
Now sample extended to support on board p3t1755 temperature sensor
on frdm_rw612 board.

Signed-off-by: Dipak Shetty <shetty.dipak@gmx.com>

dipakgmx committed:
samples: sensor: thermometer: updated to support frdm_rw612

Extended to support on board p3t1755 temperature sensor
on frdm_rw612 board.

Signed-off-by: Dipak Shetty <shetty.dipak@gmx.com>

MaureenHelm commented on samples/sensor/thermometer/sample.yaml:
```diff
@@ -41,3 +41,17 @@ tests:
       - platform:mimxrt1180_evk/mimxrt1189/cm33:SHIELD=p3t1755dp_ard_i3c
       - platform:mimxrt1180_evk/mimxrt1189/cm7:SHIELD=p3t1755dp_ard_i3c
       - platform:frdm_mcxn236/mcxn236:SHIELD=p3t1755dp_ard_i3c
+  sample.sensor.thermometer_i2c:
+    tags: sensors
+    harness_config:
+      fixture: sensor_ambient_temp_i2c
+      type: multi_line
+      regex:
+        - "Thermometer Example (.*)"
+        - "Temperature device is 0x[0-9|a-z]+, name is [a-z|0-9]+@[a-z|0-9]+"
+        - "Temperature is [0-9|.]+°C"
+    integration_platforms:
+      - frdm_rw612 # p3t1755
+    depends_on:
+      - i2c
+    filter: dt_alias_exists("ambient-temp0")
```
This new configuration isn't needed, you can add `frdm_rw612` to the list of integration platforms in the `sample.sensor.thermometer` configuration.

dipakgmx replied:
Fixed

kartben reviewed (changes requested):
Actually why would the sensor be defined in an overlay?
It should be in the board dts to be readily available to anyone interested in using it

kartben commented:
> Actually why would the sensor be defined in an overlay?
> It should be in the board dts to be readily available to anyone interested in using it 

Ping @dipakgmx - making sure you saw this :)

dipakgmx commented:
> > Actually why would the sensor be defined in an overlay?
> > It should be in the board dts to be readily available to anyone interested in using it
> 
> Ping @dipakgmx - making sure you saw this :)

Hey Benjamin, sorry. Missed this comment. I followed the frdm_mcxn947 board as the reference. @mmahadevan108, does it make sense to add this sensor here, or on the board dts?

mmahadevan108 commented:
@kartben comment makes sense. I can work on a separate PR to move this to the boards folder for MCXN947.

dipakgmx committed:
boards: nxp: frdm_rw612: add ambient temperature sensor

Now added on board p3t1755 temperature sensor node and the
corresponding alias.

Signed-off-by: Dipak Shetty <shetty.dipak@gmx.com>

dipakgmx commented:
@kartben Now moved the sensor finally to the board. Could you confirm the last commit is necessary, wherein the board was added to the sensor.yaml file?
